compiler/q-implant/tests/gen_test_data.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
```

Comment:
is this file still valid?

Reply:
There is an issue in using h5 format.

tensors can have / in their names, but h5 does not allow / in the object name. So we have to replace it into another character, which makes hard to recover the original tensor name.

---
Copied from https://github.com/Samsung/ONE/pull/11579#discussion_r1345284188

Reply:
> is this file still valid?

This file is not currently in use. 
However, we are currently applying the test data generation logic used in this file to each operator in the q-implant-op-level-test. 

(example : https://github.com/Samsung/ONE/pull/11579/files#diff-ca5cd73ec6907ee13c19d243d655c556a0d399187a8239df7c3d7e422d2c07ee)